images/breeze/output-commands-hash.txt

```diff
@@ -1,7 +1,7 @@
 # This file is automatically generated by pre-commit. If you have a conflict with this file
```

Comment:
All changes in this file are generated through `breeze setup regenerate-command-images`

Reply:
Yep.